packages/manager/apps/web/client/app/hosting/ftp/FTP.html

```diff
@@ -10,6 +10,19 @@
 
             <p data-translate="hosting_tab_FTP_dashboard_information"></p>
 
+            <div class="mb-4">
+                <span class="fa fa-life-ring mr-2" aria-hidden="true"></span>
+                <span data-translate="hosting_tab_FTP_guide_help"></span>
+                <a
+                    data-ng-href="{{:: ctrlHostingTabFTP.guide}}"
+                    target="_blank"
+                    title="{{hosting_guide_help}} {{ 'common_newtab' | translate }}"
```

Comment:
```suggestion
                    data-ng-href="{{:: ctrlHostingTabFTP.guide}}"
                    target="_blank"
                    title="{{hosting_guide_help}} {{ 'common_newtab' | translate }}"
                    rel="noopener"
```

Reply:
Hello,

thanks for the review. What's the advantage to have both target="_blank" and rel="noopener" ?

Reply:
Hello, it is for security reason. When you click on a link, which opens a new tab (as you did by setting `target="_blank")`, by default the opened tab keeps a reference to the source window. This adds vulnerability as the opened new tab may manipulate data corresponding to the original document through the object `Window.opener`.

For more information, take a look at https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

Reply:
Thanks for the info, pull request is updated with the change requested